tests/test_np.py

```diff
@@ -167,7 +170,7 @@ def test_dump_np_scalars():
 			)),
 		),
 	]
-	replaced = encode_scalars_inplace(deepcopy(data))
+	replaced = deepcopy(data)
```

Comment:
^^^^^^^^^^^^^^^^^^^^^^^^^^

This is the breaking change that this introduces.

Reply:
the only way i could think of doing this without totally breaking things was to add an extra key. sorry about this.

Reply:
please see revised decoder.